torchrl/envs/model_based/common.py

```diff
@@ -118,7 +118,7 @@ def __init__(
         batch_size: Optional[torch.Size] = None,
     ):
         super(ModelBasedEnvBase, self).__init__(
-            device=device, dtype=dtype, batch_size=batch_size
+            device=device, dtype=dtype, batch_size=batch_size, run_type_checks=False
```

Comment:
Let's make this an art that defaults to false no?

Reply:
an art?

Reply:
I guess you meant i put it as an attribute in the init